forge-gui/res/cardsfolder/upcoming/eriette_the_beguiler.txt

```diff
@@ -0,0 +1,11 @@
+Name:Eriette, the Beguiler
+ManaCost:1 W U B
+Types:Legendary Creature Human Warlock
+PT:4/4
+K:Lifelink 
+T:Mode$ Attached | ValidSource$ Aura | ValidTarget$ Permanent.nonLand+OppCtrl+cmcLEX | TriggerZones$ Battlefield | Execute$ TrigGainControl | TriggerDescription$ Whenever an Aura you control becomes attached to a nonland permanent an opponent controls with mana value less than or equal to that Aura's mana value, gain control of that permanent for as long as that Aura is attached to it.
+SVar:TrigGainControl:DB$ GainControl | NewController$ You | Defined$ Targeted | Duration$ UntilUnattached
```

Comment:
This is not working atm. don't know how to make it work without engine changes

Reply:
Thanks seem to be working now 
![image](https://github.com/Card-Forge/forge/assets/67333662/a748ce0c-2337-47fb-80dd-546ecb4cc588)